src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,34 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+from z3c.form import interfaces
+ 
+from zope import schema
+from zope.interface import Interface
+ 
+from zope.i18nmessageid import MessageFactory
+ 
+_ = MessageFactory('experimental.safe_html_transform')
+ 
+ 
+class IExperimenatalSafe_Html_TransformSettings(Interface):
+"""Global safe_html_transform settings. This describes records stored in the
+	configuration registry and obtainable via plone.registry.
+"""
+ 
+    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
```

Comment:
You should basically just copy the code from this pull request into our new package: https://github.com/plone/Products.CMFPlone/commit/ac98b0ea4ae27f4b74f4d3c5cf100b54f8acbfb7